local/compose/down.go

```diff
@@ -35,6 +35,10 @@ import (
 )
 
 func (s *composeService) Down(ctx context.Context, projectName string, options compose.DownOptions) error {
+	ctx, err := s.getUpdatedContainersStateContext(ctx, projectName)
```

Comment:
Ah please do not add more stored magical things in context, for #1304 we wanted to add an *explicit* parameter `containers` to projectFromContainerLabels(), and load containers once before invoking [projectFromContainerLabels](https://github.com/docker/compose-cli/blob/d3dbc90fdeb2f4fdcd48e1a1c711d5bb58c2c960/local/compose/down.go#L42). 
For the inspect part I have not checked the details, but I'd also rather not cache magical things in the context and pass explicit parameters.